flow/project.py

```diff
@@ -342,20 +342,20 @@ class FlowOperation(object):
 
     Any FlowOperation is associated with a specific command, which should be
     a function of :py:class:`~signac.contrib.job.Job`. The command (cmd) can
-    be stated as function, either by using str-substitution based on a job's
+    be stated as function, either by using string substitution based on a job's
     attributes, or by providing a unary callable, which expects an instance
     of job as its first and only positional argument.
```

Comment:
I just realized how confusing this wording is with regard to the command string option. I'd suggest
```suggestion
    Any FlowOperation is associated with a specific command which should be
    a function of :py:class:`~signac.contrib.job.Job`. The command (cmd) may
    either be a unary callable that expects an instance of
    :class:`~signac.contrib.job.Job` as its only positional argument and returns
    a string containing valid shell commands, or the string of commands itself.
    In either case, the resulting string may contain any attributes of the job placed
    in curly braces, which will then be substituted by Python string formatting.
```

Reply:
I don't think this is actually recognizing the commit as changing multiple lines, unfortunately. It's highlighting all the relevant lines, but I'm not sure it will actually overwrite anything other than the last one.

Reply:
You're right -- the suggestions feature isn't compatible with multi-line edits, in my understanding. I'll apply this manually.